server/src/main/java/org/opensearch/index/IndexSettings.java

```diff
@@ -658,6 +659,22 @@ public static IndexMergePolicy fromString(String text) {
         Property.Dynamic
     );
 
+    public static final Setting<Boolean> INDEX_DOC_ID_FUZZY_SET_ENABLED_SETTING = Setting.boolSetting(
```

Comment:
My thought process to name the settings like this was mostly to share that the otpimization being applied is with fuzzy set introduced on top of existing data structure. If we introduce any other postings/optimization, the name may be too generic, unless we may want to apply all together (which may or may not be true for all use cases)

That said, the suggested name sounds better from conveying what we intend to do with this setting, and can do this change if the above does not feel like a concern.

Reply:
I've renamed it do `index.optimize_doc_id_lookup.fuzzy_set.enabled` so that we've a generic setting namespace for optimize doc id which can be implemented later as well.